XamlControlsGallery/ControlExample.xaml.cs

```diff
@@ -350,5 +367,188 @@ private void OnValueChanged(ControlExampleSubstitution sender, object e)
         {
             GenerateAllSyntaxHighlightedContent();
         }
+
+        private void ScreenshotButton_Click(object sender, RoutedEventArgs e)
+        {
+            TakeScreenshot();
+        }
+
+        private void ScreenshotDelayButton_Click(object sender, RoutedEventArgs e)
+        {
+            TakeScreenshotWithDelay();
+        }
+
+        private async void TakeScreenshot()

```

Comment:
How about moving this onto a separate helper class, to which we pass the UIElement? This is not really logic related to the ControlExample, so it might be better to have it in a separate class.

Reply:
I think it's fine -- the screenshot controls are part of the ControlExample and there's not that much of it anyway, and no one else is using it.

Reply:
Fair enough.